liquibase-couchbase/src/test/java/system/precondition/SqlCheckPreconditionSystemTest.java

```diff
@@ -0,0 +1,74 @@
+package system.precondition;
+
+import com.couchbase.client.java.Collection;
+import common.operators.TestCollectionOperator;
+import liquibase.Liquibase;
+import liquibase.exception.LiquibaseException;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import system.LiquibaseSystemTest;
+
+import static common.constants.ChangeLogSampleFilePaths.SQL_CHECK_FAILED_PRECONDITION;
+import static common.constants.ChangeLogSampleFilePaths.SQL_CHECK_PRECONDITION;
+import static common.constants.TestConstants.TEST_BUCKET;
+import static common.constants.TestConstants.TEST_COLLECTION;
+import static common.constants.TestConstants.TEST_SCOPE;
+import static common.matchers.CouchbaseCollectionAssert.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class SqlCheckPreconditionSystemTest extends LiquibaseSystemTest {
+    private static final Collection collection = bucketOperator.getCollection(TEST_COLLECTION, TEST_SCOPE);
+    private static final TestCollectionOperator collectionOperator = new TestCollectionOperator(collection);
+    private static final String DOCUMENT_ID = "sqlCheckIndexTestPreconditionId1";
+
+    @BeforeAll
+    static void setUp() {
+        collection.queryIndexes().createPrimaryIndex();
+        collectionOperator.removeAllDocuments(cluster.bucket(TEST_BUCKET).scope(TEST_SCOPE));
```

Comment:
Here is a bit strange line, which makes me re-check the method signature and code. See another comment above in `collectionOperator`